File: 2023-2024/sourcecode/mod3/Lab7/ViewModelBasicsJava/app/src/main/java/com/example/viewmodelbasicsjava/ui/fragment/WordViewHolder.java
package com.example.viewmodelbasicsjava.ui.fragment;

import androidx.recyclerview.widget.RecyclerView;

import com.example.viewmodelbasicsjava.databinding.RvItemBinding;

public class WordViewHolder extends RecyclerView.ViewHolder {
    private final RvItemBinding binding;

    public WordViewHolder(RvItemBinding binding) {
        super(binding.getRoot());
        this.binding = binding;
    }

    public void bind(String item){
        binding.wordTextView.setText(item);
    }
}

File: 2023-2024/sourcecode/mod2/Lab7/ViewModelBasicsKotlin/app/src/main/java/com/example/viewmodelbasics_kotlin/ui/fragment/WordViewHolder.kt
package com.example.viewmodelbasics_kotlin.ui.fragment

import androidx.recyclerview.widget.RecyclerView
import com.example.viewmodelbasics_kotlin.databinding.RvItemBinding

class WordViewHolder(private val binding: RvItemBinding) : RecyclerView.ViewHolder(binding.root) {
    fun bind(item: String) {
        binding.wordTextView.text = item
    }
}